test.sh

```diff
@@ -2,25 +2,26 @@
 
 # this script is specified in .circleci/config.yml
 # to run as the 'tests' task
-#
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-cd $DIR
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" # get dir containing this script
+cd $DIR # always from from script dir
 
 function runTests() {
-
-
   if [[ "${CI}x" ==  "x" ]] ; then
+    # run locally
     TESTS=$({ \
-      glide nv | grep -v Utilities | grep -v longTest | grep -v peerTest | grep -v simTest; \
-      #ls simTest/*_test.go;\
-      #ls peerTest/*_test.go;\
+      glide nv | grep -v Utilities | grep -v longTest | grep -v peerTest | grep -v simTest | grep -v elections | grep -v activations | grep -v netTest | grep "..." ; \
```

Comment:
Why exclude elections tests? again grep "..." does not match just ...

Reply:
I copied what I found there originally where elections were excluded